### ISSUE TITLE
Add Serie B - Brazil

### ISSUE THREAD

**Issue by ghost** [richecr/PySoccer#9]
**Add Ligue 1 leaderboard information**

- Leaderboard URL: https://www.transfermarkt.com.br/campeonato-brasileiro-serie-b/tabelle/wettbewerb/BRA2/saison_id/2019

- Leaguue URL: https://www.transfermarkt.com.br/serie-b/startseite/wettbewerb/BRA2

**Comment by tanweeralii** [comment]
Hey I would like to contribute to this issue

**Comment by ghost** [comment]
Hello, all contributions are welcome.

Can I put you in charge of this issue?

**Comment by tanweeralii** [comment]
yes and thanks

**Comment by tanweeralii** [comment]
Hey, Can you please brief about what to do? Since the websites are in Portuguese I could not understand much. Can you please help me out? When visited the site https://www.transfermarkt.com.br/ , I could not find the list from where you are scrapping. Since the list which I am getting from running teste.py is different than that in the website. Thanks in advance:)

**Comment by ghost** [comment]
Hi, I can. You have an option to change the site to one of the allowed languages ​​as well:

![Captura de tela de 2020-10-03 17-50-09](https://user-images.githubusercontent.com/25726888/95001503-f77cf580-05a0-11eb-98a2-695da4bb6fb9.png)

On the issue, you will have to create the methods that will return these urls (in this file) and also add them to the urls dictionary. And with these changes you should already be able to obtain the information (from the classification). The main URL where the scrapping will act is this: https://www.transfermarkt.com.br/campeon-brasileiro-serie-b/tabelle/wettbewerb/BRA2/saison_id/2019. The second url I am adding, but it is not yet being used.

**Comment by ghost** [comment]
To help you more about this issue, a commit has already been made on such a task: https://github.com/richecr/PySoccer/commit/569d0d27c0668259da425eee2347a578981f16c2

The changes are the same. If you haver any questions, feel free to ask!

**Comment by tanweeralii** [comment]
Nice. Thanks

**Comment by ghost** [comment]
Hello @tanweeralii, are you having any problems/doubts with this issue?

**Comment by tanweeralii** [comment]
Yeah I will surely get back to this project tomorrow. My tests are going on and tomorrow is also a test. I am excited to contribute and learn from this project. There are some doubts by the way. Thanks

**Comment by ghost** [comment]
Okay, no problem. Any questions just ask.

**Comment by ghost** [comment]
How is the progress of this issue, @tanweeralii? Do you still intend to continue with the development?

**Comment by tanweeralii** [comment]
I am working on two projects Sanscript winter of code and dwoc, again I am not getting enough time to contribute to this issue. I am sorry, if you want you can unassign me this issue, if I will get time I will try to give a pr or I will contanct you through email.
